controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/TaskExceptions.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.server.eventProcessor.requesthandlers;
+
+import io.pravega.controller.retryable.RetryableException;
+
+public class TaskExceptions {
+    public static class StreamTaskException extends RuntimeException {
+        private static final long serialVersionUID = 1L;
+    }
+
+    public static class StartException extends StreamTaskException {
+    }
+
+    public static class EventPostException extends StreamTaskException implements RetryableException {
```

Comment:
`EventPost` does not make a lot of sense, should we say instead `PostEvent`?

Reply:
done